wordpress-mobile/WordPress-iOS
# Update WordPressUI pod to 1.5.2-beta.1

**This PR:**
- updates the podfile from WordPressUI 1.5.1 -> 1.5.2-beta.1
- the updated WordPressUI pod supports Swift 5
- updates the podfile.lock checksums for WordPressUI
- updates the shared pod schemes to version 11.3

**To test:**
Mainly we want to ensure the pod installs correctly and updates to 1.5.2-beta.1
- `rake dependencies` or `bundle exec pod install`
- (optional) build the app

PR submission checklist:
